server/vulnerabilities/cve.go

```diff
@@ -152,6 +165,10 @@ func checkCVEs(ctx context.Context, ds fleet.Datastore, logger kitlog.Logger, cp
 						if err != nil {
 							level.Error(logger).Log("cpe processing", "error", "err", err)
 						}
+
+						// TODO(mna): if CVE is within 2 days of its published date, and
+						// webhook is enabled, collect the CVE and its matching CPEs. How
+						// to get the CVE's published date is still TBD at this time.
```

Comment:
Assuming no one raises concerns over adding the `software_cpe.cpe` index, this is the main blocker for me - how to get that CVE's published date.

Reply:
Following Tomas' comment here: https://github.com/fleetdm/fleet/issues/3050#issuecomment-1026158753, I tested it with the cast and it works fine, as show above.